Postman_Sort/Postman_sort.c

```diff
@@ -0,0 +1,173 @@
+#include <stdio.h>
+
+
+int array[100], array1[100];
+int i, j, temp, max, count, maxdigits = 0, c = 0;
+
+/* Function to arrange the of sequence having same base */
+void arrange(int k,int n)
+{
+
+    for (i = k; i < n - 1; i++)
+    {
+        for (j = i + 1; j < n; j++)
+        {
+            if (array1[i] > array1[j])
+            {
+
+                temp = array1[i];          // swapping of elements
+                array1[i] = array1[j];
+                array1[j] = temp;
+
+
+                temp = (array[i] % 10);       // swapping of elements after reducing length of digits
+                array[i] = (array[j] % 10);
+                array[j] = temp;
+
+            }
+
+        }
+
+    }
+
+}
+
+
+
+int main()
+{
+
+    int t1, t, n = 1;
+    printf("Enter the size of array : ");
+    scanf("%d", &count);  // size of array
+
+    printf("Enter the elements of array : ");
+    for (i = 0; i < count; i++)
+
+    {
+
```

Comment:
Remove so many new lines.

Reply:
Okk I will do the suggested changes and make a pr! !!